### ISSUE TITLE
Missing index on ip4log.ip causes queries to perform table scans

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#3293]
Some of the ip4log queries aren't scope by tenant causing the non-usage of an index causing enormous load on the DB

**Comment by julsemaan** [comment]
Issue seems to be from pfdns which isn't tenant aware:
```
go/coredns/plugin/pfdns/pfdns.go:       pf.IP4log, err = pf.Db.Prepare("select mac from ip4log where ip = ? ")
```

**Comment by julsemaan** [comment]
Whatever we decide in #3292 can be applied here

**Comment by jrouzierinverse** [comment]
Fixed by b89a8f4eff15f5434a924cd00a23fb832c8eda4d
